bin/GameIndex.dbf

```diff
@@ -1171,10 +1171,10 @@ Name   = Dead or Alive 2 - Hardcore
 Region = PAL-M5
 Compat = 1
 [patches = 7A51F86E]
-//Patched by Prafull
-//Some missing sounds
-patch=0,EE,002b4c44,word,24060000
-patch=0,EE,002b4dc0,word,10000014
+	comment=Patch by Prafull
+	// This patch cause some missing sounds issues.
```

Comment:
If it causes sound issues, and doesn't fix anything, don't add it. ;)

Reply:
If you remove the patch the game won't work. :D Missing sounds is the side effect of the patch :P

Reply:
@prafullpcsx2 Does the game not boot at all without this patch or does it stop working elsewhere?

Reply:
The game won't boot.

https://forums.pcsx2.net/Thread-Dead-or-Alive-2-Hardcore-no-proper-sound